docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+
+  mysql-old:
+    image: mysql:5.7.27
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpass
+      MYSQL_DATABASE: springbootdb
+    ports:
+      - "3308:3306"
```

Comment:
docker-compose file must be placed on docker provision folder (provision/docker for example)